pages/api/token/temporary.ts

```diff
@@ -37,7 +37,7 @@ const handler = async (req: NextApiRequest, res: NextApiResponse, { formID }: Mi
     await sendTemporaryTokenByEmail(email, temporaryToken);
 
     logMessage.info(`Temporary Token Requested: Form ID: ${formID} Email: ${email}`);
-    res.status(200).json({ message: "success" });
+    res.status(200).json({ message: "success", formID: formID });
```

Comment:
I can't see where you need the `formID` in the rest of the pull request.

Reply:
@craigzour  - You're right; this was put in before I figured out what Bryan had put together for the Credentials Provider, and is not a part of this PR.